lib/__tests__/kv.test.js

```diff
@@ -79,4 +80,22 @@ describe('kv', () => {
     const kv = new KeyValueStore()
     expect(() => { kv.delete('hello') }).toThrowError(new Error('HTTP DELETE request failed: 404 Not Found'))
   })
+
+  test('KeyValueStore persistence', async () => {
```

Comment:
For async tests, [accept a callback function](https://github.com/dollarshaveclub/cloudworker/blob/29b36611fcb211dfadfecf995e62f78c40406f06/lib/__tests__/cloudworker-e2e.test.js#L39) as a parameter and [invoke it after the test assertion(s](https://github.com/dollarshaveclub/cloudworker/blob/29b36611fcb211dfadfecf995e62f78c40406f06/lib/__tests__/cloudworker-e2e.test.js#L55)

Reply:
Done this also